TheOtherRoles/CustomOptionHolder.cs

```diff
@@ -408,8 +409,9 @@ public class CustomOptionHolder {
             mayorTasksNeededToSeeVoteColors = CustomOption.Create(82, Types.Crewmate, "Completed Tasks Needed To See Vote Colors", 5f, 0f, 20f, 1f, mayorCanSeeVoteColors);
             mayorMeetingButton = CustomOption.Create(83, Types.Crewmate, "Mobile Emergency Button", true, mayorSpawnRate);
             mayorMaxRemoteMeetings = CustomOption.Create(84, Types.Crewmate, "Number Of Remote Meetings", 1f, 1f, 5f, 1f, mayorMeetingButton);
-
+            
             engineerSpawnRate = CustomOption.Create(90, Types.Crewmate, cs(Engineer.color, "Engineer"), rates, null, true);
+            engineerFixPer = CustomOption.Create(7000, Types.Crewmate,"Sabotages Fixes Per", new string[] { "Round", "Game" }, engineerSpawnRate);
```

Comment:
```suggestion
            engineerFixPer = CustomOption.Create(7000, Types.Crewmate, "Sabotages Fixes Per", new string[] { "Round", "Game" }, engineerSpawnRate);
```
As another minor stylistic nit, spaces after commas.